packages/blinks-core/src/api/BlinkConfig.ts

```diff
@@ -1,65 +1,65 @@
 import { Connection } from '@solana/web3.js';
 import {
-  type Action,
+  type BlinkInstance,
   AbstractActionComponent,
   DEFAULT_SUPPORTED_BLOCKCHAIN_IDS,
 } from './Action';
 import type { SignMessageData } from './actions-spec.ts';
 
-export interface ActionContext {
+export interface BlinkExecutionContext {
   originalUrl: string;
-  action: Action;
+  action: BlinkInstance;
   actionType: 'trusted' | 'malicious' | 'unknown';
   triggeredLinkedAction: AbstractActionComponent;
 }
 
-export interface IncomingActionConfig {
+export interface IncomingBlinkConfig {
   rpcUrl: string;
-  adapter: Pick<ActionAdapter, 'connect' | 'signTransaction' | 'signMessage'> &
-    Partial<Pick<ActionAdapter, 'metadata'>>;
+  adapter: Pick<BlinkAdapter, 'connect' | 'signTransaction' | 'signMessage'> &
+    Partial<Pick<BlinkAdapter, 'metadata'>>;
 }
 
 /**
- * Metadata for an action adapter.
+ * Metadata for an blink adapter.
  *
  * @property supportedBlockchainIds List of CAIP-2 blockchain IDs the adapter supports.
  *
  * @see {BlockchainIds}
  */
-export interface ActionAdapterMetadata {
+export interface BlinkAdapterMetadata {
   /**
    * List of CAIP-2 blockchain IDs the adapter supports.
    */
   supportedBlockchainIds: string[];
 }
 
-export interface ActionAdapter {
-  metadata: ActionAdapterMetadata;
-  connect: (context: ActionContext) => Promise<string | null>;
+export interface BlinkAdapter {
+  metadata: BlinkAdapterMetadata;
+  connect: (context: BlinkExecutionContext) => Promise<string | null>;
   signTransaction: (
     tx: string,
-    context: ActionContext,
+    context: BlinkExecutionContext,
   ) => Promise<{ signature: string } | { error: string }>;
   confirmTransaction: (
     signature: string,
-    context: ActionContext,
+    context: BlinkExecutionContext,
   ) => Promise<void>;
   signMessage: (
     data: string | SignMessageData,
-    context: ActionContext,
+    context: BlinkExecutionContext,
   ) => Promise<{ signature: string } | { error: string }>;
 }
 
-export class ActionConfig implements ActionAdapter {
+export class BlinkConfig implements BlinkAdapter {
```

Comment:
should we name it `BlinkSolanaConfig`? technically it's only for solana..

Reply:
Good point! Will do

Reply:
done